source/badjs.js

```diff
@@ -0,0 +1,4 @@
+let bobby = fetch(
```

Comment:
Codacy has a fix for the issue: ['bobby' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/cse110-fa21-group27/cse110-fa21-group27/pullRequest?prid=8413591)
```suggestion
const bobby = fetch(
```